### DISCUSSION TITLE
Does the vat_id field have to be unique?

### DISCUSSION THREAD

**Discussion by pmuch78** [YetiForceCompany/YetiForceCRM#16963] [How to]
Hello.
As we are making business with local government, we have many clients with the same vat_id. Is it possible to add them as different clients in Yeti?

**Comment by rskrzypczak** [comment]
I do not go into the reasonableness of this approach, but you can disable the duplicate vat_id verifications in the administration panel: _Software configuration/System tools/Event handlers_

**Reply by pmuch78** [reply]
Polish law isn't always reasonable.
If you're interested:
https://www.fakturaxl.pl/faktura-dla-gminy-jednostki-samorzadu
